backend/impl/db.go

```diff
@@ -34,47 +40,36 @@ func validateResult(r sql.Result, e error) error {
 	return nil
 }
 
-func updateUser(u UserArgs) error {
+type sqlDB struct {
```

Comment:
If we don't make DB function methods, we won't need double wrapping of DB object. *sql.DB will be a member of handler type.

Reply:
I'd say it gives a good point about the function area. Gin handlers and db functions have very similar names. sqlDB receiver and handler receiver allow clearly separate them from each other. db.db isn't a big problem there.